getting-started/value-vs-data-binding.md

```diff
@@ -28,23 +28,23 @@ In this documentation, we will refer to Value Binding as the process of passing
 
 Value binding has two forms:
 
-* **one-way binding** (syntax like `<ParameterName>="@someField"`, for example `Value="@someField"`, `Page="@someField"`).
```

Comment:
Avoid bold for emphasis. You can add a reference to the corresponding heading - this is also a form of emphasis.